Add Purinta vaults (#312)
api3dao/api3-docs: docs/curation/index.md

## Vaults

> Api3 operates five vaults across Ethereum mainnet and the Robinhood chain.

Reviewer: ```suggestion
Api3 operates multiple vaults across Ethereum mainnet and the Robinhood chain.
```

The Api3 dCOMP vault supplies the USDC borrow-side liquidity for this market, making the mechanism viable.

### Purinta

Purinta is a pair of meme token lending vaults curated by Api3: a USDC vault on Ethereum mainnet and a USDG vault on the Robinhood chain.
Each supplies the borrow-side liquidity for a set of isolated meme token markets — PEPE, SPX6900 and SHIB on Ethereum, and CASHCAT on Robinhood.

Meme tokens are more volatile and higher-risk than blue-chip collateral, so a market is only listed when its token has enough on-chain liquidity to support healthy swaps and liquidations. Supply caps and LLTVs are configured conservatively to limit that exposure.

These markets are surfaced to borrowers through [Purinta app](https://purinta.xyz), a meme token lending interface.

### Accessing the vaults

> All five vaults can be viewed and accessed through the Morpho app, where Api3 operates as a [verified curator.](https://forum.morpho.org/c/vaults/api3/57) Live figures - total deposits, APY, enabled markets, and current supply caps - are shown there.

Reviewer: ```suggestion
Each of the vaults can be also accessed through the Morpho app, where Api3 operates as a [verified curator.](https://forum.morpho.org/c/vaults/api3/57) Live figures - total deposits, APY, enabled markets, and current supply caps - are shown there.
```
